src/assets/jui/components/timeline/_timeline.scss

```diff
@@ -53,6 +53,7 @@
   color: $govuk-secondary-text-colour;
   display: inline;
   margin: 0;
+  margin-left: govuk-spacing(1);
```

Comment:
Unsure how this file exists ... surely the scss goes in src/app/domain/case-viewer/components/timeline-panel/timeline-panel.component.scss ... please could we move it.

Reply:
^ please move this file ....